### DISCUSSION TITLE
Image OCR index using vision model

### DISCUSSION THREAD

**Discussion by vccler** [langgenius/dify#5642] [Suggestion]
some pptx,docx,pdf documents might have images embeded inside. Could we discuss about the implementation of parsing the image via vision model to extract to text format and index into vector database. In this way, it will provide better answer when user ask question about the images.

**Comment by guchenhe** [comment]
working on this as part of our RAG pipeline restructure!

**Comment by indracha-maersk** [comment]
Is there an update on this? I have PDF with images and want to use vision to fetch images as a part of answers
